python/utils/__init__.py

```diff
@@ -1,5 +1,9 @@
 from utils import colorText, str2bool, find_moose_executable, runExe, check_configuration
-from MooseYaml import MooseYaml
+
+try:
+    from MooseYaml import MooseYaml
+except:
+    pass
```

Comment:
I know we do this in other places but what happens if it's not there? Do you get an error when you use a utility that requires the Yaml library?